### PR TITLE
Modified signal to activate deactivate a user on nodebb  [#371]

### PR DESCRIPTION
**Pull request by rosheen33**
connects #371

### CODE REVIEW

File: common/djangoapps/nodebb/signals/handlers.py [hunk 1]
```diff
@@ -92,19 +93,23 @@ def create_user_on_nodebb(sender, instance, created, **kwargs):
...
-@receiver(pre_save, sender=User, dispatch_uid='activate_user_on_nodebb')
-def activate_user_on_nodebb(sender, instance, **kwargs):
+@receiver(pre_save, sender=User, dispatch_uid='activate_deactivate_user_on_nodebb')
+def activate_deactivate_user_on_nodebb(sender, instance, **kwargs):
     """
-    Activate a user on nodebb whenever user become active on edx platform
+    Activate or Deactivate a user on nodebb whenever user's active state changes on edx platform
     """
-    if instance.is_active and User.objects.filter(pk=instance.pk, is_active=False).exists():
+    current_user_obj = User.objects.filter(pk=instance.pk).first()
```
Comment (muhammadnadeem) [inline common/djangoapps/nodebb/signals/handlers.py]: Why we are calling .first method here? I think it's not possible that we have user duplication

Comment (rosheen33) [reply common/djangoapps/nodebb/signals/handlers.py]: we are getting based on primary key. So It will be just one entry or None.

Comment (rosheen33) [reply common/djangoapps/nodebb/signals/handlers.py]: I have done it so that we don't have to use try except blocks.

Comment (muhammadnadeem) [reply common/djangoapps/nodebb/signals/handlers.py]: I think this we should not use .first as we are not using current_user_obj anywhere except in conditions. We don't need the first object, we just need to check if user exists. That can achieved just using User.objects.filter(pk=instance.pk)

Comment (rosheen33) [reply common/djangoapps/nodebb/signals/handlers.py]: we need it's is_active field

**Pull request merged**